Binary/Loader/FileSystemLoader.php

```diff
@@ -38,30 +38,48 @@ class FileSystemLoader implements LoaderInterface
     /**
      * @param MimeTypeGuesserInterface  $mimeGuesser
      * @param ExtensionGuesserInterface $extensionGuesser
-     * @param string[]                  $dataRoots
-     * @param LocatorInterface          $locator
+     * @param string[]|LocatorInterface $locator
      */
```

Comment:
Ahh, very good point and I didn't think of that.  A better idea is adding a comment, or maybe an `@see` referencing this PR or the docs added in this PR?